proposals/2161/registry.md

```diff
@@ -110,6 +116,62 @@ airflow/
 	facets/AirflowRunFacet.json
 ```
 ```
+core/
+	CODEOWNERS
+	registry.json
+	{
+		producer: {
+			root_doc_URL: "https://openlineage.io/spec/facets/",
+			sample_URL: "https://github.com/OpenLineage/OpenLineage/tree/main/spec/tests/",
+			facets: [
+				"ColumnLineageDatasetFacet.json": {
+					"owner": "core",
+					"spec_versions": [ "1-0-1", "1-0-0" ],
```

Comment:
What syntax will spec versions support? Is there one list entry per version or would there be support for =, <, <= prefixes? @Sheeri

Reply:
Great question! I see now that it probably should be using semantic versioning, this is already a solved problem.
https://github.com/npm/node-semver

I will change examples to:
"spec_versions": ">=1.0.0"
and
"spec_versions": "=1.0.0"

(I'm also not sure why I used the separator as - instead of . but I will change that too.)
We don't need to account for Advanced stuff like ~ or ^.

Reply:
That makes sense. In this case I meant `since` but I can see how we'd want to remove facets as well. 

Another thought I have is that we could derive this information from git history. So we could also not duplicate it in the file.